java/app/grapheneos/setupwizard/data/WelcomeData.kt

```diff
@@ -7,6 +7,8 @@ import java.util.Locale
 
 object WelcomeData : ViewModel() {
     val selectedLanguage = MutableLiveData<Locale>()
+    val oemUnlocked = MutableLiveData<Boolean>() // bootloader unlocked
```

Comment:
`oemUnlocked` is not the same as bootloader being unlocked

Reply:
the api doc seems to say so, although a bit ambiguous; so I've been waiting for test results on real device (don't have a test device at hand).
are you sure this is not the same API we want?

Reply:
I've made a mistake here, oemUnlocked indeed indicates whether the bootloader is unlocked. Got confused by OemLockManager method names.